packages/hint-axe/src/util/axe.ts

```diff
@@ -24,6 +24,26 @@ type Registration = {
 
 type RegistrationMap = Map<EngineKey, Map<string, Registration[]>>;
 
+let html: HTMLElement;
```

Comment:
@molant we'll need to reset `html` on `scan::end` or this won't work correctly on subsequent runs. 

That said, what do you think about enhancing the `can-evaluate::script` instead to include the relevant `HTMLDocument` instance to make this simpler?

Reply:
Need to find out if that's possible in jsdom or if we will have to emit the event at another time but definitely better than what is there right now.

Reply:
Your registration thing in axe is giving me a headache @antross 😅

Reply:
ready for another review